code/modules/projectiles/projectile/special.dm

```diff
@@ -325,10 +325,10 @@
 	name = "sonic bolt"
 	icon_state = "energy2"
 	damage_types = list(BRUTE = 10)
-	armor_penetration = 90 // It is a sound-wave
+	armor_penetration = 20 // It is a sound-wave liquifing organs I guess
 	kill_count = 7
 	check_armour = ARMOR_ENERGY
-	var/golem_damage_bonus = 20 // Damage multiplier against ameridians.
+	var/golem_damage_bonus = 90 // Damage multiplier against ameridians.
```

Comment:
The damage bonus is multiplicative : At 10 damage & a bonus of 90, that's a total of 900 damage, or enough to even 1-shot a Behemoth Golem! With the old 20, it dealt 200 damage, enough to one-shot regular golems, but needing 3 shoots for the behemoths